protocol/messenger.go

```diff
@@ -502,7 +502,7 @@ func (m *Messenger) Init() error {
 			continue
 		}
 		switch chat.ChatType {
-		case ChatTypePublic, ChatTypeProfile:
+		case ChatTypePublic, ChatTypeProfile, ChatTypeTimeline:
```

Comment:
This is used to create filters, do you need a filter for a timeline chat?

Reply:
no thanks